migrations/schema.sql

```diff
@@ -12,4 +12,4 @@ CREATE TABLE IF NOT EXISTS gue_jobs
   updated_at  TIMESTAMPTZ NOT NULL
 );
 
-CREATE INDEX IF NOT EXISTS idx_gue_jobs_selector ON gue_jobs (queue, run_at, priority);
+CREATE INDEX IF NOT EXISTS idx_gue_jobs_selector ON gue_jobs (queue, job_type, run_at, priority);
```

Comment:
this is backward-compatibility breaking change actually

Reply:
Why is it a breaking change? if the index exists it will ignore the line.
Do you want me to add an additional index for this?

Reply:
because w/out this change new functionality may cause performance issues, and given that `gue` is just a library - the app using it may start experiencing serious issues, e.g. running out of available DB connections because gue worker queries are taking too long

previously all DB changes were part of the major version update

I'm fine to let this index change in the minor version update, but then you need to prepare a migration that can be mentioned in the change notes